.gitignore

```diff
@@ -139,4 +139,7 @@ __bentoml_tmp_sdist_build
 helm/YataiService/charts
 
 # Integration test data
-tests/integration/data
\ No newline at end of file
+tests/integration/data
+
+# Emacs buffers
+*~
```

Comment:
This change is unrelated to adding EvalML support. I am an emacs user and find this helpful.